### PR TITLE
chore: use jenkins org if running on trusted

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -70,24 +70,25 @@ def getJavaVersion(path) {
...
-  if (BRANCH_NAME == 'master') {
-    tags << "jenkins4eval/openjdk:${jdkShortVersion}-${type}-${jdkType}-windowsservercore-ltsc2019"  
-    tags << "jenkins4eval/openjdk:${jdkLongVersion}-${type}-${jdkType}-windowsservercore-ltsc2019"
+  def org = "${isTrusted()?'jenkins':'jenkins4eval'}"  
+  if (env.BRANCH_NAME == 'master') {
+    tags << "${org}/openjdk:${jdkShortVersion}-${type}-${jdkType}-windowsservercore-ltsc2019"  
+    tags << "${org}/openjdk:${jdkLongVersion}-${type}-${jdkType}-windowsservercore-ltsc2019"
     if (jdkShortVersion == '15') {
-      tags << "jenkins4eval/openjdk:${type}-${jdkType}-windowsservercore-ltsc2019"
+      tags << "${org}/openjdk:${type}-${jdkType}-windowsservercore-ltsc2019"
```
Comment (olblak) [inline Jenkinsfile]: You can use the env variable `DOCKERHUB_ORGANIZATION` as defined here 
https://github.com/jenkins-infra/pipeline-library/blob/f982f7b16670a91d086d3dc0dba68a476a4b061c/vars/infra.groovy#L22

Comment (olblak) [reply Jenkinsfile]: it's easier to maintain in long term than putting this logic here

Comment (garethjevans) [reply Jenkinsfile]: done

Comment (garethjevans) [reply Jenkinsfile]: good spot

**Review by olblak — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**